docs/task/task.md

```diff
@@ -104,6 +104,32 @@ class DispatcherMixin:
         return cls.tcp_port
 ```
 
+## Netmiko Show Running Config Command
+
+The Netmiko `show_command` tells Netmiko which command to use to get the config, generally used to backup the configuration. You can override the default provided based on this logic:
+
+- Prefer `obj.cf["config_command"]` if it is a valid string.
+- Prefer `obj.get_config_context()["config_command"]` if it is a valid string.
+- Use the default command defined in your Netmiko dispatcher, often defaulting to `NetmikoDefault` which sets `show run`.
+
+Here is the implementation:
+
+```python
+class NetmikoDefault(DispatcherMixin):
+
+    config_command = "show run"
+
+    @classmethod
+    def _get_config_command(cls, obj) -> str:
+        custom_field = obj.cf.get("config_command")
+        if isinstance(custom_field, str):
+            return custom_field
```

Comment:
Would this return an empty value?
What is the behaviour for custom field that is given a value and then value is erase/deleted ?

Reply:
thanks!